neptune/new/sync/sync.py

```diff
@@ -41,6 +38,7 @@
 from neptune.new.internal.disk_queue import DiskQueue
 from neptune.new.internal.id_formats import QualifiedName, UniqueId
 from neptune.new.internal.operation import Operation
+from neptune.new.internal.utils.debug_log import logger
```

Comment:
I think we should made a shortcut in `neptune.new.internal.utils.__init__` to be able to import it with `from neptune.new.internal.utils import logger` + `debug_log` looks weird as it was already used for run URL echo.